CHANGES.rst

```diff
@@ -1,10 +1,11 @@
 0.20.0 (unreleased)
 -------------------
 
-other
-^^^^^
+- Improve documentation (part 1) [#483]
 
-- Replace ``pkg_resources`` with ``importlib.metadata``. [#478]
+- other
+^^^^^
+  - Replace ``pkg_resources`` with ``importlib.metadata``. [#478]
```

Comment:
Good question with regard to 'other', which does not belong. I've removed it. The next line shows as a change due to a change in the indentation I believe. It otherwise was already present in the main branch.